packages/frontend/src/components/common/GlobalAlert.js

```diff
@@ -224,11 +224,10 @@ const GlobalAlertNew = ({ globalAlert, actionStatus, clearGlobalAlert, closeIcon
                                                     {noTranslationFound
                                                         ? <Translate id={`reduxActions.default.${alert.success ? 'success' : 'error'}`} />
                                                         : <SafeTranslate id={alert.messageCode} data={alert.data} />
-                                                        // : <Translate id={alert.messageCode} data={escapeHtml(alert.data)} />
                                                     }
                                                     {!alert.success &&
                                                         <a
-                                                            href={(noTranslationFound || msgCode.includes('Sorry an error has occured')) 
+                                                            href={(noTranslationFound || msgCode.includes('Sorry an error has occurred'))
```

Comment:
This should be safe, `Sorry an error has occured` doesn't show up in any translation files with the typo.